Style.css

```diff
@@ -0,0 +1,7 @@
+h1 {
+    color: rgb(0, 255, 115);
+    font-size: 85px;
+    font-family: 'Courier New', Courier, monospace, Geneva, Tahoma, sans-serif;
+    text-align: center;
+  }
```

Comment:
- [ ] The class name that was given to your h1 in the HTML file should be used here to style the content

E.G:
```CSS
.header {
    color: rgb(0, 255, 115);
    font-size: 85px;
    font-family: 'Courier New', Courier, monospace, Geneva, Tahoma, sans-serif;
    text-align: center;
  }
```

With this you will be sure you follow the project requirement